app/src/main/java/cm/aptoide/pt/DeepLinkAnalytics.java

```diff
@@ -72,10 +74,18 @@ public void websiteFromAppViewWebPage() {
     websiteSourceGroup(SOURCE_GROUP_OPTION_APP_VIEW);
   }
 
+  public void websiteFromBundlesWebPage() {
+    websiteSourceGroup(SOURCE_GROUP_OPTION_BUNDLES);
+  }
+
   public void websiteFromStoreWebPage() {
     websiteSourceGroup(SOURCE_GROUP_OPTION_STORE);
   }
 
+  public void websiteFromThankYouWebPage() {
+    websiteSourceGroup(SOURCE_GROUP_OPTION_THANK_YOU);
+  }
+
   public void newUpdatesNotification() {
     analyticsManager.logEvent(createMap(SOURCE, NEW_UPDATES_NOTIFICATION), APP_LAUNCH,
         AnalyticsManager.Action.AUTO, getViewName(true));
```

Comment:
Not related to what you did but the timelineNotification() method here is never used;

Reply:
done